Document what region is used by default (#832)
zeit/docs: pages/docs/v2/deployments/configuration.mdx

**Valid values:** a list of valid [region identifiers](/docs/v2/platform/regions-and-providers).

> By setting and modifying this value, you can decide the deployment **regions** of the [Lambdas](/docs/v2/deployments/concepts/lambdas) that get created as a result of the [build steps](/docs/v2/deployments/builds). By default, the closest region is used, or SFO when deploying via Now for GitHub.

Author: Our deployments done by Now for GitHub seem to go to SFO, could anyone confirm?

Also, does this apply to GitLab as well? I'd be happy to add it to docs if it's true.

Reviewer: Hmm, you could be right, there. I didn't think about the fact that GitHub won't have access to DNS information for the committer. cc @arunoda Should we document that SFO is the default for GitHub/Gitlab, or suggest that the user sets a region before anything.

Reviewer 2: It's default for now. But it can be changed.
It's always to set it manually.

> By setting and modifying this value, you can decide the deployment **regions** of the [Lambdas](/docs/v2/deployments/concepts/lambdas) that get created as a result of the [build steps](/docs/v2/deployments/builds). By default, the closest region is used, or SFO when deploying via Now for GitHub.

Reviewer: ```suggestion
By setting and modifying this value, you can decide the deployment **regions** of the [Lambdas](/docs/v2/deployments/concepts/lambdas) that get created as a result of the [build steps](/docs/v2/deployments/builds). By default, the closest region to the geographical origin of the deployment is used.
```

Reviewer: Amended my suggestion to factor in the git integration comments.

Author: Would there be a way to avoid the word "origin" here? It often means the datacenter where lambdas run. Also, is "Now for Git" an officially used term? I think I've seen "Now for GitHub" and "Now for GitLab".

Reviewer: "Geographical location" instead?

Author: 👍 , I've pushed the change.

Do you also want me to update the "Now for Git" part or is it good as it is?
